src/wrapped_radix_engine_toolkit/high_level/manifest_building/manifests_create_tokens.rs

```diff
@@ -221,7 +221,7 @@ impl TransactionManifest {
 
 impl TokenDefinitionMetadata {
     pub(crate) fn for_nft_collection(index: U11) -> Self {
-        let word = bip39_word_by_index(index).word;
+        let word = bip39_word_by_index(index.clone()).word;
```

Comment:
U11 *was* `Copy` but not anymore, we wanna have more control over it, since it contains 1/24 of a mnemonic (for 24 words mnemonics).